### ISSUE TITLE
PyPi installation fails on macOS Mojave

### ISSUE THREAD

**Issue by rensmacneill** [peterwittek/somoclu#136]
Hi there, 

I have tried to install somoclu with PyPi on macOS Mojave using

`pip install somoclu`

however I receive the following error when attempting to work my way through your example:
`Warning: the binary library cannot be imported. You cannot train maps, but you can load and analyze ones that you have already saved.
If you installed Somoclu with pip on macOS, this typically means missing a linked library. If you compiled Somoclu with GCC, please make sure you have set DYLD_LIBRARY_PATH to include the GCC path. For more information, please refer to the documentation.`

I then tried the alternative of compiling from source using

- git clone https://github.com/peterwittek/somoclu.git
- cd somoclu
- ./autogen.sh
- ./configure
- make python
- cd src/Python
- sudo python setup.py install`

However this fails as well with `clang: error: unsupported option '-fopenmp’`

I then tried various suggestions as found here: [https://github.com/peterwittek/somoclu/issues/90](url) though none have helped. 

If you could point me in the right direction to solve this, it would be much appreciated. 

Kind regards, 

Rens MacNeill

**Comment by xgdgsc** [comment]
Does the conda forge version work?

**Comment by rensmacneill** [comment]
Thanks for the suggestion. I just tested the conda-forge version and there are no issues importing or using the package.

Are there any plans to update the PyPi version for macOS Mojave?

**Comment by TyLeRRR** [comment]
> Does the conda forge version work?

No. is any chance to expect a normal installation via pip? Or this project is abandoned, and nobody is at home?

**Comment by peterwittek** [comment]
We have a lack of developers with experience on said platform.
